web/templates/dialog_widget.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="micromodal" id="{{modal_unique_id}}" aria-hidden="true">
-    <div class="modal__overlay" tabindex="-1">
+    <div class="modal__overlay" tabindex="-1" {{#unless close_on_overlay_click}}data-ignore-overlay-click{{/unless}}>
```

Comment:
This is a weird name; usually `data-` properties should have a value. I think just calling this `ignore-overlay-click` would be correct.

Reply:
Made it `ignore-overlay-click`

btw, we have things like `data-micromodal-close`, `data-simplebar`. Maybe we can update those post 10.0